lib/recaptcha/helpers.rb

```diff
@@ -205,6 +211,27 @@ def self.to_message(_key, default)
       HTML
     end
 
+    private_class_method def self.recaptcha_v3_onload_script(site_key, action, callback, id, options = {})
+      nonce = options[:nonce]
+      nonce_attr = " nonce='#{nonce}'" if nonce
+
+      <<-HTML
+        <script#{nonce_attr}>
+          // Define function so that we can call it again later if we need to reset it
+          // This executes reCAPTCHA and then calls our callback.
+          function #{recaptcha_v3_execute_function_name(action)}() {
+            grecaptcha.ready(function() {
+              grecaptcha.execute('#{site_key}', {action: '#{action}'}).then(function(token) {
+                //console.log('#{id}', token)
```

Comment:
remove ?

Reply:
as  this gets executed after the callback we must not remove it

Reply:
I meant the console.log :)

Reply:
sure. This is still in the original code too. I'll remove it in both cases